kinto/core/storage/generators.py

```diff
@@ -51,9 +51,5 @@ class UUID4(Generator):
     duplicate would be about 50% (`source <http://en.wikipedia.org/wiki/\
 Universally_unique_identifier#Random_UUID_probability_of_duplicates>`_).
     """
-    regexp = (r'^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-'
```

Comment:
I don't really like that change :(

Shouldn't we actually inherit from the kinto-core UUID4 generator in kinto to override the regexp with a more general one?